src/event_loop.rs

```diff
@@ -253,6 +253,20 @@ impl<T> EventLoopWindowTarget<T> {
   pub fn cursor_position(&self) -> Result<PhysicalPosition<f64>, ExternalError> {
     self.p.cursor_position()
   }
+
+  /// Sets the progress bar state
+  /// 
+  /// ## Platform-specific
+  /// 
+  /// - **Windows:** Use the Progress Bar Function Available in Window (Windows can have different progress bars for different window)
+  /// - **macOS:** Unimplemented.
+  /// - **iOS / Android:** Unsupported.
+  #[inline]
+  #[allow(unused)]
+  pub fn set_progress_bar(&self, progress: ProgressBarState) {
+    #[cfg(target_os = "macos")]
```

Comment:
Should this be Windows and Linux only?

Reply:
Yes if u like to impl macos one using dock
Or else impl it as an alias to macos dock impl